### DISCUSSION TITLE
Partially loaded webiste on GitHub Pages

### DISCUSSION THREAD

**Discussion by UT-99** [community/community#36307] [Other Feature Feedback, Questions, & Ideas]
Hi Team, 
Just to confirm one concern about GitHub pages deployment, I deployed my website on git hub pages as soon i received my URL for the hosted application, the website was loaded partially not completely means the background images and the content relative to the browsers window was not loaded at same time and it took after 1 min to load to whole website properly after deployment. 

So, its expected or some issue is linked to my website deployment. 

Pasting the hosted application link:
https://ut-99.github.io/CSS_6/

**Comment by yoannchaudet** [comment]
At deployment time we issue a request to invalidate our CDN's cache. As the CDN is distributed and maintain cache all across the planet, it can take a few minutes for all assets to clear up globally. So yes this is expected and unlikely to be an issue with your site specifically.

**Reply by UT-99** [reply]
Thanks for update on this, it's really helpful

**Comment by ossanna16** [comment]
Hey there @UT-99 and thank you for asking a good question! I'm glad you found our community 🙂
